dev-python/uri_template/metadata.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
```

Comment:
```suggestion
	<maintainer type="person" proxied="yes> 
```

See here for the details: https://www.gentoo.org/glep/glep-0067.html#package-maintenance

Reply:
@AndrewAmmerlaan ok.